modules/nf-core/bbmap/sendsketch/main.nf

```diff
@@ -0,0 +1,35 @@
+process BBMAP_SENDSKETCH {
+    tag "$file"
+    label 'process_low'
+
+    conda "bioconda::bbmap=39.01"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bbmap:39.01--h5c4e2a8_0':
+        'quay.io/biocontainers/bbmap:39.01--h5c4e2a8_0' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), path('results.txt')  , emit: hits
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def file_used = file.size() > 1 ? file[0] : file
+    """    
+    sendsketch.sh \\
+        in=${file_used} \\
+        out='results.txt' \\
+        $args \\
+        -Xmx${task.memory.toGiga()}g
```

Comment:
It's better to add a check and see if the memory has been set (see this module for an example: https://github.com/nf-core/modules/blob/master/modules/nf-core/gatk4/combinegvcfs/main.nf)